### PR TITLE
Update create-a-shared-mailbox.md

### PR DESCRIPTION
**Pull request by msbemba**
added steps to disable Automapping

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/email/create-a-shared-mailbox.md [hunk 1]
```diff
@@ -292,6 +292,14 @@ A Group in Outlook is like a shared mailbox. For a comparison of the two, see [C
 Automapping is set on the user's mailbox, not the shared mailbox.
 
 This means if you try to use a security group to manage who has access to the shared mailbox, automapping won't work. So, if you want automapping, you have to assign permissions explicitly.
+
+ By default, Automapping is enabled. The admin's can disable the automapping feature via powershell while granting access to the user.
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/create-a-shared-mailbox.md]:
```suggestion
 Automapping is enabled by default. Admins can disable the automapping feature via PowerShell while granting access to the user.
```

File: Office365-Admin/email/create-a-shared-mailbox.md [hunk 2]
```diff
@@ -292,6 +292,14 @@ A Group in Outlook is like a shared mailbox. For a comparison of the two, see [C
...
+
+Run the following commands:
```
Comment (JohanFreelancer9) [inline Office365-Admin/email/create-a-shared-mailbox.md]:
```suggestion
Run the following commands:
```
```suggestion
Run the following command:
```

File: Office365-Admin/email/create-a-shared-mailbox.md [hunk 3]
```diff
@@ -292,6 +292,14 @@ A Group in Outlook is like a shared mailbox. For a comparison of the two, see [C
 Automapping is set on the user's mailbox, not the shared mailbox.
 
 This means if you try to use a security group to manage who has access to the shared mailbox, automapping won't work. So, if you want automapping, you have to assign permissions explicitly.
+
+ Automapping is enabled by default. Admins can disable the automapping feature via PowerShell while granting access to the user.
```
Comment (cmcatee-MSFT) [inline Office365-Admin/email/create-a-shared-mailbox.md]: Delete these additions and replace with:

Automapping is on by default. To learn how to turn it off, see [Remove automapping for a shared mailbox](https://docs.microsoft.com/en-us/office365/troubleshoot/administration/remove-automapping-for-shared-mailbox).

Comment (mypil) [reply Office365-Admin/email/create-a-shared-mailbox.md]: @msbemba - please apply Cern's recommended changes to this PR. Thank you.

**Comment by mypil** [comment]
@cmcatee-MSFT - PR is ready for final review following your recommended changes. Please assist to merge. Thank you.

**Pull request merged**